CRM/Core/Config.php

```diff
@@ -295,6 +294,10 @@ public function cleanupCaches($sessionReset = FALSE) {
     CRM_Core_DAO_AllCoreTables::flush();
     CRM_Utils_System::flushCache();
 
+    // note this used to be earlier, but was crashing because of api4 instability
+    // during extension install
+    UserJob::delete(FALSE)->addWhere('expires_date', '<', 'now')->execute();
```

Comment:
@ufundo IMO we are just asking for trouble by calling the api from within super low-level functions like this one. I would suggest calling the BAO directly for this instead.